boards/xtensa/m5stack_stamps3/doc/index.rst

```diff
@@ -0,0 +1,199 @@
+| 3   | ``m5stack_stamps3_clkout0`` - Channel 0 |     |                                 |
++-----+-----------------------------------------+-----+---------------------------------+
+| 4   |                                         |     |                                 |
++-----+-----------------------------------------+-----+---------------------------------+
+| 5   | ``m5stack_stamps3_spilcd`` - MOSI       |     |                                 |
++-----+-----------------------------------------+-----+---------------------------------+
+| 6   | ``m5stack_stamps3_spilcd`` - CLK        |     |                                 |
++-----+-----------------------------------------+-----+---------------------------------+
+| 7   | ``m5stack_stamps3_spilcd`` - CS         | 28  | **3V3**                         |
++-----+-----------------------------------------+-----+---------------------------------+
+| 8   |                                         | 27  | ``m5stack_stamps3_uart0`` - TXD |
++-----+-----------------------------------------+-----+---------------------------------+
+| 9   | ``m5stack_stamps3_clkout0`` - Channel 2 | 26  |                                 |
++-----+-----------------------------------------+-----+---------------------------------+
+| 10  |                                         | 25  | ``m5stack_stamps3_uart0`` - RXD |
++-----+-----------------------------------------+-----+---------------------------------+
+| 11  | **GND**                                 | 24  |                                 |
++-----+-----------------------------------------+-----+---------------------------------+
+| 12  | ``m5stack_stamps3_i2c1`` - SDA          | 23  |                                 |
++-----+-----------------------------------------+-----+---------------------------------+
+| 13  | **5V**                                  | 22  | **EN**                          |
++-----+-----------------------------------------+-----+---------------------------------+
+| 14  | ``m5stack_stamps3_i2c1`` - SCL          | 21  |                                 |
++-----+-----------------------------------------+-----+---------------------------------+
+| 15  | ``m5stack_stamps3_i2c0`` - SDA          | 20  |                                 |
++-----+-----------------------------------------+-----+---------------------------------+
+| 16  |                                         | 19  |                                 |
++-----+-----------------------------------------+-----+---------------------------------+
+| 17  | ``m5stack_stamps3_i2c0`` - SCL          | 18  | **GND**                         |
++-----+-----------------------------------------+-----+---------------------------------+
+
+Power supply
+============
+
+M5Stack-StampS3 requires a single 5V input power supply. The module internally
+featues a DCDC (MUN3CAD01-SC) to generate the 3.3V needed for the MCU.
```

Comment:
```suggestion
features a DCDC (MUN3CAD01-SC) to generate the 3.3V needed for the MCU.
```

Reply:
Fixed in [38f0a59](https://github.com/zephyrproject-rtos/zephyr/commit/38f0a59c9701f755f1b41c0489c642aeb2e0bc7b)